package.json

```diff
@@ -189,8 +189,8 @@
     "parcel-bundler": "1.12.3",
     "parcel-plugin-css-to-string": "^2.5.2",
     "prettier": "^2.2.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "19.0.0",
+    "react-dom": "19.0.0",
```

Comment:
Let's make it ^ so that it would be possile to use newer versions in development if needed.